test/TestInterfaceFunctionSelectors.t.sol

```diff
@@ -2,27 +2,15 @@
 pragma solidity ^0.8.15;
 
 import "forge-std/Test.sol";
-import {IERC7540Deposit, IERC7540Redeem, IERC7540DepositReceiver, IERC7540RedeemReceiver} from "src/interfaces/IERC7540.sol";
+import {IERC7540Deposit, IERC7540Redeem} from "src/interfaces/IERC7540.sol";
 
 contract TestInterfaceFunctionSelectors is Test {
-  
-
     // --- erc165 checks ---
     function testERC165InterfaceSelectors() public {
-        bytes4 erc7540Deposit = 0x1683f250;
-        bytes4 erc7540Redeem = 0x0899cb0b;
+        bytes4 erc7540Deposit = 0xce3bbe50;
+        bytes4 erc7540Redeem = 0x620ee8e4;
 
```

Comment:
Can you add that both shoudl return `true` on `0xe3bc4e65` and `0x2f0a18c5`?

https://github.com/centrifuge/liquidity-pools/blob/main/test/unit/ERC7540Vault.t.sol#L91 and https://github.com/centrifuge/liquidity-pools/blob/main/test/unit/ERC7540Vault.t.sol#L92